### PR TITLE
fix: connection string to env

### LINKED ISSUE #2: Connection-String to env
[here](https://github.com/kaiwalyakoparkar/practice-node-applications/blob/1564e6b730b2f0b5649896480875e94493aade42/routes/server.js#L14) your connection-string for mongo is hard-coded. I'd put this to your `.env`

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #2

### CODE REVIEW

File: routes/server.js [hunk 1]
```diff
@@ -1,90 +1,99 @@
...
+const home = require("../views/home.hbs");
+const signin = require("../views/signin.hbs");
+const update = require("../views/update.hbs");
+const unsub = require("../views/unsub.hbs");
+const status = require("../views/status.hbs");
 
 //Mongoose connections and configs
 //Connecting to mongo db
-mongoose.connect('mongodb://localhost:27017/Newsletter', {useNewUrlParser: true, useUnifiedTopology: true});
+const mongoConnection = process.env.MONGO_CONNECTION_STRING;
```
Comment (Cahllagerfeld) [inline routes/server.js:14]: Connectionstring gets read from env

File: routes/server.js [hunk 2]
```diff
@@ -1,90 +1,99 @@
...
+
+mongoose.connect(mongoConnection || "mongodb://localhost:27017/Newsletter", {
```
Comment (Cahllagerfeld) [inline routes/server.js:16]: in case mongoConnection is `undefined` your app takes the local mongo connection

**Review by Cahllagerfeld — COMMENTED** [review]
Here are my changes

**Review by kaiwalyakoparkar — APPROVED** [review]
Thanks :) That looks neat. I would use same approach to connect with mongodb now! :+1:

**Pull request merged**